application_form/tests/test_lottery_haso_services.py

```diff
@@ -274,13 +274,12 @@ def test_winning_cancels_lower_priority_applications_if_not_reserved(
 
 
 @mark.django_db
-def test_winning_does_not_cancel_lower_priority_apartments_if_reserved(
+def test_winning_cancel_lower_priority_apartments_if_reserved(
     elastic_haso_project_with_5_apartments,
 ):
     # If an application wins an apartment but has already won a different apartment
-    # with a lower priority, then we should not automatically cancel the reserved
-    # application despite it being lower priority. This kind of situation needs to
-    # be handled manually by the salesperson.
+    # with a lower priority, then we should automatically cancel the reserved
+    # application despite it being lower priority.
```

Comment:
```suggestion
    # application.
```
The despite part doesn't really make sense anymore.